miden-lib/src/transaction/inputs.rs

```diff
@@ -94,6 +94,7 @@ fn extend_advice_inputs(
     add_chain_mmr_to_advice_inputs(tx_inputs.block_chain(), advice_inputs);
     add_account_to_advice_inputs(tx_inputs.account(), tx_inputs.account_seed(), advice_inputs);
     add_input_notes_to_advice_inputs(tx_inputs.input_notes(), tx_args, advice_inputs);
+    advice_inputs.extend_map(tx_args.get_advice_map().clone());
     add_tx_script_inputs_to_advice_map(tx_args.tx_script(), advice_inputs);
```

Comment:
In the future, if we put tx script arguments into the advice map of `TransactionArgs`, we could get rid of `add_tx_script_inputs_to_advice_map()` function here.